lenovo/thinkpad/l13-yoga/default.nix

```diff
@@ -0,0 +1,15 @@
+{ nixos, lib, pkgs, config, stdenv, ... }:
+{
+  imports = [
+    ../.
+    ../../../common/cpu/intel
+    ../../../common/pc/laptop/acpi_call.nix
+  ];
+
+  services.throttled.enable = lib.mkDefault true;
+
+  # automatic screen orientation
+  hardware.sensor.iio.enable = true;
+
+  services.xserver.wacom.enable = lib.mkDefault config.services.xserver.enable;
```

Comment:
Please move this to `../yoga.nix` and import this here and in `x13-yoga` instead.

Reply:
I think I'll only move `hardware.sensor…`, since there is no guaranty all thinkpad-yogas will have a wacom touchscreen.

Reply:
ok. Let's keep it for now.